README.md

```diff
@@ -91,3 +92,7 @@ We're happy to incorporate all improvements to JPlag into this code base. Feel f
 
 ### Adding new languages
 Adding a new language frontend is quite simple. Have a look at one of the `jplag.frontend` projects. All you need is a parser for the language (e.g., for ANTLR or for JavaCC) and a few lines of code that sends the tokens (that are generated by the parser) to JPlag.
+
+
+## License
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FDavidVarga1995%2Fjplag.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FDavidVarga1995%2Fjplag?ref=badge_large)
```

Comment:
Codacy has a fix for the issue: [Files should end with a single newline character](https://app.codacy.com/gh/DavidVarga1995/jplag/pullRequest?prid=8510190)
```suggestion
[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FDavidVarga1995%2Fjplag.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FDavidVarga1995%2Fjplag?ref=badge_large)

```